lib/core/Cargo.toml

```diff
@@ -26,32 +25,30 @@ flutter_rust_bridge = { version = "=2.8.0", features = [
 ], optional = true }
 log = { workspace = true }
 lwk_common = "0.8.0"
-lwk_signer = "0.8.0"
-lwk_wollet = { git = "https://github.com/breez/lwk", branch = "breez-sdk-liquid-0.6.3" }
-#lwk_wollet = "0.8.0"
+lwk_signer = { version = "0.8.0", default-features = false }
 rusqlite = { version = "0.31", features = ["backup", "bundled"] }
+tokio = { version = "1", default-features = false, features = ["rt", "macros"] }
+# TODO: Change on top of main once PR is merged
+sdk-common = { git = "https://github.com/breez/breez-sdk", rev = "19ed955c6fa854ad6cd2beb2eca9127d8099c506", features = ["liquid"] }
 rusqlite_migration = "1.0"
-sdk-common = { git = "https://github.com/breez/breez-sdk", rev = "84578d9abe1dd0d59867f93a22a92220ab16a52e", features = ["liquid"] }
 serde = { version = "1.0.197", features = ["derive"] }
 serde_json = "1.0.116"
 strum = "0.25"
 strum_macros = "0.25"
 thiserror = { workspace = true }
-tokio-tungstenite = { version = "0.21.0", features = ["native-tls-vendored"] }
-openssl = { version = "0.10", features = ["vendored"] }
-tokio = { version = "1", features = ["rt", "macros"] }
-tokio-stream = { version = "0.1.14", features = ["sync"] }
 url = "2.5.0"
 futures-util = { version = "0.3.28", default-features = false, features = [
     "sink",
     "std",
 ] }
-async-trait = "0.1.80"
+async-trait = "0.1.86"
 hex = "0.4"
-reqwest = { version = "=0.11.20", features = ["json"] }
-electrum-client = { version = "0.21.0", default-features = false, features = [
-    "use-rustls-ring",
-    "proxy",
+reqwest = { version = "0.12", default-features = false, features = [
+    "charset",
+    "http2",
+    "json",
+    "blocking",
```

Comment:
Seems like the WASM-compatible version of reqwest is the async one: https://docs.rs/reqwest/latest/reqwest/#wasm